VERSION

```diff
@@ -1 +1 @@
-v2.0.14
\ No newline at end of file
+v2.0.16
```

Comment:
OT: can we add the VERSION to gitignore? It;s autogenerated anyway

Reply:
sure, I can do that in this PR